bench/wtperf/wtperf.c

```diff
@@ -1341,7 +1341,7 @@ checkpoint_worker(void *arg)
             if (stop || wtperf->error)
                 break;
         }
-        /* If tiered storage is enabled we want a final checkpoint. */
+        /* If tiered storage is disabled we are done. Otherwise, we want a final checkpoint. */
         if (wtperf->error || (stop && opts->tiered_flush_interval == 0))
```

Comment:
I think the way the code is structured now, we just want to do:
```
 if (wtperf->error || stop)
    break;
```
We don't need a comment (since it would be wrong).